### ISSUE TITLE
Literature Review - Qwen-VL

### ISSUE THREAD

**Issue by ToluClassics** [The-African-Research-Collective/newspaper-parser#3]
https://arxiv.org/abs/2308.12966

**Comment by dugerij** [comment]
> https://arxiv.org/abs/2308.12966

Native LLM are purely text-based and are often found lacking in their applications to other modalities. These have led to research in LVLM, but current open-source LVLM suffer from inadequate training and optimisation and as a result lack the ability to execute fine-grained perception such as object grounding and text reading. This limits their applications to real-world visual scenarios.

The paper presents Qwen-VL series which are built using as a foundation Qwen-LM coupled with with a visual encoder (a ViT) and a positional-aware adapter. The adapter is made up of a single-layer cross-attention module initialised randomly and uses a group of trainable vectors(embeddings) as query vectors and image features from the visual encoder as keys for cross attention operations. 2D positional embeddings are also incorporated into the query-key pairs to mitigate loss of positional details during compression.
Inputs to the models have special tokens to denote images and text. Similarly outputs from the model have special tokens that identify bounding boxes and allow 'referral' of content marked in bounding boxes.

The model is trained in a 3 steps(2 pre-training and 1 fine-tuning step), First a pre-training step during which the Qwen-LM is frozen and the other two parts are trained or low resolution image-text pairs. Secondly, all models are trained on high resolution Multi-task and interleaved VL data (This produces the Qwen-VL model). The final step, freezes the Visual encoder and fine-tunes the Qwen-LM and the positional aware encoder on chat interleaved VL data making it suitable for instruction following and dialogue capabilities(This produces the Qwen-VL-Chat).
